deno_webgpu/01_webgpu.js

```diff
@@ -1877,12 +1810,7 @@ class GPUDevice extends EventTarget {
         "OperationError",
       );
     }
-    const operations = SafePromiseAll(scope.operations);
-    return PromisePrototypeThen(
-      operations,
-      () => PromiseResolve(null),
-      (err) => PromiseResolve(err),
-    );
+    return PromiseResolve(scope.errors[0] ?? null);
```

Comment:
The spec states:
> Let error be any one of the items in scope.[[[errors]]](https://gpuweb.github.io/gpuweb/#dom-gpu-error-scope-errors-slot), or null if there are none.
 >For any two errors E1 and E2 in the list, if E2 was caused by E1, E2 should not be the one selected.
 
 it would be neat if we could use an AggregateError here instead, but i guess currently not possible. might be worth bringing up to webgpu spec.

Reply:
I don't really understand this, but would be worth filing an issue with more context if you think this is weird.